pkg/client/client_test.go

```diff
@@ -239,6 +239,7 @@ func generateRuntimeConfig(bundlePath, rootfs string) error {
 		return err
 	}
 	g.SetProcessCwd("/")
+	g.SetProcessTerminal(true)
```

Comment:
it would be cool if we could test it fails if SetProcessTerminal(false) but SetTerminal(true) and vice versa. if it doesn't fail we don't have to look that hard into it for now

Reply:
Added a new test case for the terminal, so now we have two :+1: